src/products/controllers/products/products.controller.ts

```diff
@@ -5,52 +5,29 @@ import {
     HttpStatus,
     Param,
     Post,
-    Patch,
     Res,
+    Put,
   } from '@nestjs/common';
-  import { ProductsService } from 'src/products/services/products/products.service';
-
-  
+import { ProductsService } from 'src/products/services/products/products.service';
+import { CreateProductDto } from 'src/products/dto/create-product.dto';
+import { UpdateProductDto } from 'src/products/dto/update-product.dto';
   @Controller('products')
```

Comment:
My fault, I didn't update the Wiki correctly. I missed a couple things regarding the routes. We want to ensure routes are standard across our repositories.

I updated the Wiki with the correct endpoints. Can you update this here as well? Thank you 

## Product Controller (`/products`)

> **Note**: These endpoints need created using Controller & Service.

- `GET /products`: Fetch all products.
- `GET /products/find/:id`: Fetch a specific product by its ID.
- `POST /products/new`: Create a new product.
- `PUT /products/update/:id`: Update an existing product by its ID.
- `DELETE /products/remove/:id`: Delete a product by its ID.

## User Controller (`/user`)

> **Note**: These endpoints need created using Controller & Service.

- `POST /user/new`: Add a new user.
- `GET /user`: Fetch all users.
- `GET /user/find/:id`: Fetch a specific user by their ID.
- `GET /user/email/:email`: Fetch a user by their email.
- `PUT /user/update/:id`: Update an existing user by their ID.
- `DELETE /user/remove/:id1: Remove an existing user by their ID.

Reply:
Absolutely! I will do that. Thank you. 👍

Reply:
@theGaryLarson I went ahead and fixed the product API endpoints. I will let the PO/PM create issues with the user module, giving other students a chance to learn (user's controller, service, and DTO creation).